docs/content/documentation/getting-started/configuration.md

```diff
@@ -121,11 +117,29 @@ paths = "on"
 taxonomies = "on"
 anchors = "on"
 
-# Optional translation object. Keys should be language codes.
+# Optional translation object. The key if present should be a language code.
+# Example:
+#     default_language = "fr"
+#
+#     [translations]
+#     [translations.fr]
+#     title = "Un titre"
+#
+#     [translations.en]
+#     title = "A title"
+#
+# translated title value will be available using {{ config.get_translation("fr", "title") }} in templates
```

Comment:
I'm not sure about this.

Reply:
Probably not needed, there should be a dedicated i18n page explaining how everything works.

Reply:
@Keats, ok I'm gonna remove it. I'll be happy to help about the i18n page, but right now I don't know how it works.